crates/noirc_evaluator/src/ssa/acir_gen/internal_var.rs

```diff
@@ -70,7 +76,7 @@ impl InternalVar {
     /// Example: f(x,y) = 10
     /// `f` is a constant expression because there are no
     /// bi-variate or uni-variate terms, just a constant.
-    fn is_const_expression(&self) -> bool {
```

Comment:
Not blocking: These methods were moved to ACIR so we cn remove these in a later PR